.github/workflows/ci.yml

```diff
@@ -7,6 +7,40 @@ on:
       - master
 
 jobs:
+- job: CheckFormatting
+
+  pool:
+    vmImage: 'windows-2022'
+
+  steps:
+  - script: |
+      if NOT EXIST "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat" (
+        echo ERROR: Could not locate 'vcvars' batch file. This script is intended to be run from a build machine & exit /B 1
+      )
+      call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvars64.bat"
+
+      call find_clang_format.cmd
+      if %ERRORLEVEL% neq 0 exit /b %ERRORLEVEL%
+
+      git clang-format origin/master --binary "%CLANG_FORMAT%" --style file -- cppwinrt/*.h cppwinrt/*.cpp fast_fwd/*.h fast_fwd/*.cpp prebuild/*.h prebuild/*.cpp scratch/*.h scratch/*.cpp strings/*.h strings/*.cpp test/*.h test/*.cpp vsix/*.h vsix/*.cpp
+
+      if %ERRORLEVEL% neq 0 (
+        echo ##vso[task.logissue type=error]ERROR: This branch contains changes that have not been formatted with 'clang-format'
```

Comment:
`echo ##vso[task.logissue type=error]` probably not valid for GitHub actions(?)

Reply:
Agreed.  I think I've got it converted over to GH syntax now.